.storybook/preview.js

```diff
@@ -17,28 +17,29 @@
 /**
  * External dependencies
  */
-import React from 'react';
 import { ThemeProvider } from 'styled-components';
-import { addDecorator, addParameters } from '@storybook/react';
 import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
 import {
   theme as designSystemTheme,
   lightMode,
   ThemeGlobals,
   ModalGlobalStyle,
 } from '@googleforcreators/design-system';
-import {
-  theme,
-  GlobalStyle,
-  EditorConfigProvider,
-} from '@googleforcreators/story-editor';
 import { CropMoveableGlobalStyle } from '@googleforcreators/moveable';
-import {
-  DashboardGlobalStyle,
-  DashboardKeyboardOnlyOutline,
-  ConfigProvider as DashboardConfigProvider,
-  ApiProvider,
-} from '@googleforcreators/dashboard';
+
+/**
+ * Internal dependencies
+ */
+// Disable reason:
+// Importing from the dashboard and story editor roots break fast refresh in storybook.
+// Prevented by importing the necessary providers and configs directly.
+/* eslint-disable import/no-relative-packages */
+import { GlobalStyle as DashboardGlobalStyle } from '../packages/dashboard/src/theme';
+import DashboardKeyboardOnlyOutline from '../packages/dashboard/src/utils/keyboardOnlyOutline';
+import DashboardConfigProvider from '../packages/dashboard/src/app/config/configProvider';
+import ApiProvider from '../packages/dashboard/src/app/api/apiProvider';
+import EditorConfigProvider from '../packages/story-editor/src/app/config/configProvider';
+/* eslint-enable import/no-relative-packages */
 
```

Comment:
The story editor and dashboard stories were not fast refreshing, and were logging warnings like this:

|Dashboard|Story Editor|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/161574777-377bce0e-d146-428f-babd-addc63b83d77.png)|![image](https://user-images.githubusercontent.com/22185279/161574704-71907a3f-e9d6-4e58-9424-ec64607f166a.png)|

Importing directly from the modules here removes these warnings and fixes fast refresh.

Reply:
Interesting 🤔 

I guess storybook doesn't like npm workspaces where the packages are referenced from `node_modules` but actually symlinked.